iot/manager/manager.go

```diff
@@ -24,12 +24,10 @@ import (
 	// [END imports]
 )
 
-// Registry Management
-
-// [START iot_create_registry]
+// [START get_client]
```

Comment:
Don't have to fix this now, but region tags should start with the product they're used for (enforced by the samples tracker).